docs/api/web-contents.md

```diff
@@ -1797,6 +1798,18 @@ Returns `boolean` - Whether the devtools is opened.
 
 Returns `boolean` - Whether the devtools view is focused .
 
+#### `contents.getDevToolsTitle()`
+
+Returns `string` - the current title of the devtools window. This will only be visible
+if devtools is been open in `undocked` or `detach` mode.
+
+#### `contents.setDevToolsTitle(title)`
+
+* `title` string
+
+Changes the title of devtools to `title`.This will only be visible if devtools is
+been open in `undocked` or `detach` mode.
```

Comment:
```suggestion
Changes the title of the DevTools window to `title`. This will only be visible if DevTools is
opened in `undocked` or `detach` mode.
```